refine

```diff
@@ -4,6 +4,18 @@
 #               OpenRefine Control System             #
 ##########################################################
 
+# ---------------- user settings ------------------------
+
+if [ ! -d $HOME/.openrefine ]; then
+	echo "Creating user directory"
+	mkdir $HOME/.openrefine
```

Comment:
We currently use `.local/share/openrefine` by default - why change to `.openrefine`?